go-controller/pkg/libovsdbops/router.go

```diff
@@ -44,6 +44,428 @@ func FindLogicalRoutersWithPredicate(nbClient libovsdbclient.Client, p logicalRo
 	return found, err
 }
 
+// LOGICAL ROUTER POLICY OPs
+
+type logicalRouterPolicyPredicate func(*nbdb.LogicalRouterPolicy) bool
+
+// FindLogicalRouterPoliciesWithPredicate looks up logical router policies from
+// the cache based on a given predicate
+func FindLogicalRouterPoliciesWithPredicate(nbClient libovsdbclient.Client, p logicalRouterPolicyPredicate) ([]*nbdb.LogicalRouterPolicy, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), types.OVSDBTimeout)
+	defer cancel()
+	found := []*nbdb.LogicalRouterPolicy{}
+	err := nbClient.WhereCache(p).List(ctx, &found)
+	return found, err
+}
+
+// GetLogicalRouterPolicy looks up a logical router policy from the cache
+func GetLogicalRouterPolicy(nbClient libovsdbclient.Client, policy *nbdb.LogicalRouterPolicy) (*nbdb.LogicalRouterPolicy, error) {
+	found := []*nbdb.LogicalRouterPolicy{}
+	opModel := OperationModel{
+		Model:          policy,
+		ExistingResult: &found,
+		OnModelUpdates: nil, // no update
```

Comment:
Would it be useful to define this value as an enum/variable so we don't have to add this comment?
Same for `OnModelUpdates: []interface{}{}, // update all provided values`